common/app/common/InlineStyles.scala

```diff
@@ -118,15 +118,14 @@ object InlineStyles extends GuLogging {
     }
   }
 
-  /** I found drawing this table helpful in understanding the logic of this function.
-    * As you add styles from left to right:
+  /** I found drawing this table helpful in understanding the logic of this function. As you add styles from left to
+    * right:
     *
-    * Existing Style | New Style    | Which should win?
-    * ---------------------------------------------------
-    *  !important    |      -       | existing
-    *      -         |  !important  | new
-    *  !important    |  !important  | new
-    *      -         |      -       | new
+    * Existing Style | New Style | Which should win? --------------------------------------------------- !important | -
+    * \| existing
+    *   - \| !important | new
+    * !important | !important | new
+    *   - \| - | new
```

Comment:
I think this is less readable…

Reply:
Agree! Let me try reverting it back to what it was and see if scalafmt will complain 🤫

Reply:
Unfortunately it doesn't like it